views/js/runner/proxy/cache/proxy.js

```diff
@@ -167,62 +333,56 @@ define([
              * try to load the next items
              * @returns {Promise} that always resolves
              */
-            function loadNextItem() {
-                return new Promise(function (resolve) {
-                    var siblings = navigationHelper.getSiblingItems(self.testMap, itemIdentifier, 'both', self.cacheAmount);
-                    var missing = _.reduce(siblings, function (list, sibling) {
-                        if (!self.hasItem(sibling.id)) {
-                            list.push(sibling.id);
-                        }
-                        return list;
-                    }, []);
-
-                    //don't run a request if not needed
-                    if (missing.length) {
-                        _.delay(function () {
-                            self.request(self.configStorage.getTestActionUrl('getNextItemData'), {itemDefinition: missing})
-                                .then(function (response) {
-                                    if (response && response.items) {
-                                        _.forEach(response.items, function (item) {
-                                            if (item && item.itemIdentifier) {
-
-                                                //store the response and start caching assets
-                                                self.itemStore.set(item.itemIdentifier, item);
-
-                                                if (item.baseUrl && item.itemData && item.itemData.assets) {
-                                                    assetLoader(item.baseUrl, item.itemData.assets);
-                                                }
-                                            }
-                                        });
-                                    }
-
-                                    resolve();
-                                })
-                                .catch(resolve);
-                        }, loadNextDelay);
-                    } else {
-                        resolve();
+            var loadNextItem = function loadNextItem(){
```

Comment:
no, since it uses the `qtiServiceProxy.request` which push the request in the `promiseQueue`, so having a second wrapping promise was unecessary